src/main/java/com/google/sps/ListEventDataServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
```

Comment:
We also have a file in main for this servlet though is empty so we could copy & paste the code i the main function. (address src/main/java/com/google/sps/servlets/EventRetrivalServlet.java)

Reply:
For some reason I do not have to servlets in my files even though Im pretty sure I merged the accepted pull requests, once I figure it out I'll copy and paste it. Same goes for the CreateEvent.java